### ISSUE TITLE
Android in-app purchases error

### ISSUE THREAD

**Issue by VitaZheltyakov** [godotengine/godot#21327]
**Godot version:**
Godot 3.0.6

**OS/device including version:**
Windows

**Issue description:**
If you add android in-app purchases to the project (http://docs.godotengine.org/en/3.0/tutorials/platform/android_in_app_purchases.html), you get error:
`Failed to retrieve non-existent singleton 'GodotPayments'`

**Steps to reproduce:**
This error you can get in the [demo project](https://github.com/godotengine/godot-demo-projects/tree/master/misc/android_iap)

**Comment by volzhs** [comment]
in the demo project,
it says
```
1. Project > Project Settings
2. write [Category : android] / [Property : modules] / [Type : String] and click Add
3. Click "Android" on left panel
4. double click on right filed of "modules"
5. write "org/godotengine/godot/GodotPaymentV3"
```

did you follow this steps?

**Comment by VitaZheltyakov** [comment]
In the demo project, this setting has already been completed.
I also did these steps manually.

**Comment by volzhs** [comment]
did you test on android device? `GodotPayments` is only available on android device.

**Comment by VitaZheltyakov** [comment]
No, I did not test on android device. I do not know how to catch errors on remote devices. But in Godot version 2.1 of this error was not and everything worked fine.

**Comment by volzhs** [comment]
then use `has_singleton` to check and prevent error message.
it is not a bug or issue.

**Comment by volzhs** [comment]
i would not say it is incorrect but could be improved

**Comment by meng2468** [comment]
Just for clarification, does the error actually impact the IAP functionality?

**Comment by volzhs** [comment]
@meng2468 I don't think so.
